### DISCUSSION TITLE
generate_radi_base_and_pre-base.md Working/Outdated?

### DISCUSSION THREAD

**Discussion by izzaaaatanishq** [JdeRobot/RoboticsAcademy#2440] [General]
Hi Community Members,
I was going through the docs of Robot Academy and found a file named generate_radi_base_and_pre-base.md in docs. I am not sure whether it is still being used for generating the base image. Since it was mentioned it creates base radi for mini radi. But in the doc of mini radi we just use build.sh. So could you please tell me whether it is used till now also or its outdated?
Thamks

**Comment by pawanw17** [comment]
Yes, this does seem deprecated... @jmplaza We removed the pre-base and base Dockerfiles from RoboticsInfrastructure?

We were going to have a miniRADI for only RoboticsAcademy and a larger RADI that'd include RoboticsAcademy, Visual Circuit and other tools in the same docker image.

**Reply by izzaaaatanishq** [reply]
oh, got it
